sdk/src/impl/whirlpool-client-impl.ts

```diff
@@ -23,17 +30,69 @@ export class WhirlpoolClientImpl implements WhirlpoolClient {
     if (!account) {
       throw new Error(`Unable to fetch Whirlpool at address at ${poolAddress}`);
     }
-    const tokenInfos = await getTokenInfos(this.ctx.fetcher, account, false);
```

Comment:
I noticed this `refresh` was set to false originally. Since parts of the token mint info can change on subsequent requests, I changed the value to use the `refresh` param.

Let me know if there's something I'm missing here.